index.html

```diff
@@ -2930,6 +2931,40 @@ <h2 id="View">Backbone.View</h2>
       events that the view has <a href="#Events-listenTo">listenTo</a>'d.
     </p>
 
+    <p id="View-events">
+      <b class="header">events</b><code>view.events or view.events()</code>
+      <br />
+      The <b>events</b> hash (or function) can be used to specify DOM event listeners
```

Comment:
"can be used to specify DOM event listeners for your View" => "can be used to declaratively specify the DOM events that will be bound to methods on your View with `delegateEvents`"